javascript-forms/form-values.js

```diff
@@ -0,0 +1,17 @@
+var $elements = document.querySelector('form');
+var $button = $elements[3];
+var userName = $elements.name;
+var userEmail = $elements.email;
+var userMessage = $elements.message;
+
+function handleInput(event) {
+  var newObject = {};
+  newObject[userName.name] = userName.value;
+  newObject[userEmail.name] = userEmail.value;
+  newObject[userMessage.name] = userMessage.value;
```

Comment:
Hey Jumanah! This will work, but please revise this to make use of the `elements` property on the `form` DOM element